clojure-mode.el

```diff
@@ -1843,7 +1842,6 @@ content) are considered part of the preceding sexp."
           (if (looking-at (regexp-quote ns))
               (message "ns form is already sorted")
             (sleep-for 0.1)
-            (redisplay)
```

Comment:
I think you shoudl also kill those surrounding sleeps, as I can't see why they would be needed.

Reply:
I see one before and after the redisplay.